auth/src/main/res/layout/fui_provider_button_anonymous.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.firebase.ui.auth.util.ui.SupportVectorDrawablesButton
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/anonymous_button"
+    style="@style/FirebaseUI.Button.AccountChooser.AnonymousButton"
+    android:text="@string/fui_sign_in_anonymously">
+</com.firebase.ui.auth.util.ui.SupportVectorDrawablesButton>
```

Comment:
Nit: unnecessary tag (could just be `/>`)

Reply:
Done.